Virtual Keyboard/Assets/Scripts/CategoryManager.cs

```diff
@@ -247,4 +247,24 @@ public class CategoryManager : MonoBehaviour {
 		}
 	}
 
+	[ContextMenu("Add New Category")]
+	void addCategory() {
+		Debug.Log ("Add New Category");
+
+		GameObject newCategory = new GameObject ();
+		newCategory.transform.SetParent (transform);
```

Comment:
use newCategory.transform.SetParent(transform, false);

You can then remove the localScale,localPosistion and local rotation lines